onnxruntime/core/providers/cpu/ml/tree_ensemble_aggregator.h

```diff
@@ -13,8 +13,13 @@ namespace ml {
 namespace detail {
 
 struct TreeNodeElementId {
```

Comment:
>TreeNodeElementId { [](http://example.com/codeflow?start=7&length=19)

Would 
`using TreeNodeElementId = std::tuple<int64_t, int64_t>;`
save a lot of the code below?

Reply:
I find the code more readable by keeping tree_id and node_it but I can change it.